src/pages/game/components/chat/chat-messages.component.jsx

```diff
@@ -0,0 +1,82 @@
+import "./chat-messages.component.scss";
+import "./../../../../theme/flex.scss";
+import "./../../../../theme/theme.scss";
+import { Popover, TextField } from "@mui/material";
+import { Button } from "@mui/material";
+import { useState } from "react";
+import SendIcon from "@mui/icons-material/Send";
+
+const ChatMessages = (props) => {
+  const [message, setMessage] = useState();
+
+  const handleNewChatMessageClick = () => {
+    props.handleNewChatMessageClick(message);
+    setMessage("");
+  };
+
+  const [anchorEl, setAnchorEl] = useState(null);
+
+  const handleClick = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const open = props.isOpen;
+  const id = open ? "simple-popover" : undefined;
```

Comment:
remove all the code from line 17 to here(28). Not needed. We are already sending all the needed data from the parent.